cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -383,41 +383,98 @@ esp_err_t _esp_wifi_rx_cb(void *buffer, uint16_t len, void *eb)
+    "AUTH_LEAVE",                           /* 3 */
+    "ASSOC_EXPIRE",                         /* 4 */
+    "ASSOC_TOOMANY",                        /* 5 */
+    "NOT_AUTHED",                           /* 6 */
+    "NOT_ASSOCED",                          /* 7 */
+    "ASSOC_LEAVE",                          /* 8 */
+    "ASSOC_NOT_AUTHED",                     /* 9 */
+    "DISASSOC_PWRCAP_BAD",                  /* 10 */
+    "DISASSOC_SUPCHAN_BAD",                 /* 11 */
+    "BSS_TRANSITION_DISASSOC",              /* 12 */
+    "IE_INVALID",                           /* 13 */
+    "MIC_FAILURE",                          /* 14 */
+    "4WAY_HANDSHAKE_TIMEOUT",               /* 15 */
+    "GROUP_KEY_UPDATE_TIMEOUT",             /* 16 */
+    "IE_IN_4WAY_DIFFERS",                   /* 17 */
+    "GROUP_CIPHER_INVALID",                 /* 18 */
+    "PAIRWISE_CIPHER_INVALID",              /* 19 */
+    "AKMP_INVALID",                         /* 20 */
+    "UNSUPP_RSN_IE_VERSION",                /* 21 */
+    "INVALID_RSN_IE_CAP",                   /* 22 */
+    "802_1X_AUTH_FAILED",                   /* 23 */
+    "CIPHER_SUITE_REJECTED",                /* 24 */
+    "TDLS_PEER_UNREACHABLE",                /* 25 */
+    "TDLS_UNSPECIFIED",                     /* 26 */
+    "SSP_REQUESTED_DISASSOC",               /* 27 */
+    "NO_SSP_ROAMING_AGREEMENT",             /* 28 */
+    "BAD_CIPHER_OR_AKM",                    /* 29 */
+    "NOT_AUTHORIZED_THIS_LOCATION",         /* 30 */
+    "SERVICE_CHANGE_PERCLUDES_TS",          /* 31 */
+    "UNSPECIFIED_QOS",                      /* 32 */
+    "NOT_ENOUGH_BANDWIDTH",                 /* 33 */
+    "MISSING_ACKS",                         /* 34 */
+    "EXCEEDED_TXOP",                        /* 35 */
+    "STA_LEAVING",                          /* 36 */
+    "END_BA",                               /* 37 */
+    "UNKNOWN_BA",                           /* 38 */
+    "TIMEOUT",                              /* 39 */
+
+    "PEER_INITIATED",                       /* 46 */
+    "AP_INITIATED",                         /* 47 */
+    "INVALID_FT_ACTION_FRAME_COUNT",        /* 48 */
+    "INVALID_PMKID",                        /* 49 */
+    "INVALID_MDE",                          /* 50 */
+    "INVALID_FTE",                          /* 51 */
+
+    "TRANSMISSION_LINK_ESTABLISH_FAILED",   /* 67 */
+    "ALTERATIVE_CHANNEL_OCCUPIED",          /* 68 */
+
+    "BEACON_TIMEOUT",                       /* 200 */
+    "NO_AP_FOUND",                          /* 201 */
+    "AUTH_FAIL",                            /* 202 */
+    "ASSOC_FAIL",                           /* 203 */
+    "HANDSHAKE_TIMEOUT"                     /* 204 */
+    "CONNECTION_FAIL",                      /* 205 */
+    "AP_TSF_RESET",                         /* 206 */
+    "ROAMING",                              /* 207 */
 };
+
+struct _valid_esp_wifi_disc_reason_codes {
+    uint8_t from;
+    uint8_t to;
+} _valid_esp_wifi_disc_reason_codes_t;
```

Comment:
Ok, one more nitpicky change request. Let's better use identifiers that start with the `_esp_wifi`, i.e. `s/_valid_esp_wifi/_esp_wifi_valid`, here and below
```suggestion
struct _esp_wifi_valid_disc_reason_codes {
    uint8_t from;
    uint8_t to;
} _esp_wifi_valid_disc_reason_codes_t;
```